### PR TITLE
Add documentation of configuration for slide_local

### PR DESCRIPTION
**Pull request by dontinelli**
## Proposed change

SSIA

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/133182
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

## Summary by CodeRabbit

- **Documentation**
	- Added a new "Options" section for the Slide Local integration, detailing the "Invert position" checkbox for cover configurations.
	- Updated the existing configuration block to include the new option.

### CODE REVIEW

File: source/_integrations/slide_local.markdown [hunk 1]
```diff
@@ -47,6 +47,10 @@ password:
   type: string
 {% endconfiguration_basic %}
 
+## Configuration
```
Comment (andrewsayre) [inline source/_integrations/slide_local.markdown]: Two things:

1. This is a duplicate heading. `{% include integrations/config_flow.md %}` adds the heading "Configuration"
2. Is this meant to describe configuration options? If so, use the heading "Options" and the `{% configuration_basic %}` parameter structure.

**Review by andrewsayre — DISMISSED** [review]

File: source/_integrations/slide_local.markdown [hunk 2]
```diff
@@ -47,6 +47,15 @@ password:
   type: string
 {% endconfiguration_basic %}
 
+## Options
```
Comment (frenck) [inline source/_integrations/slide_local.markdown]:
```suggestion
{% include integrations/option_flow.md %}
```

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/slide_local.markdown [hunk 3]
```diff
@@ -47,6 +47,15 @@ password:
...
+
+{% configuration_basic %}
+Invert position:
+  description: Check the box, if your cover uses inverted positions for open and closed.
+  required: false
+  type: bool
```
Comment (klaasnicolaas) [inline source/_integrations/slide_local.markdown]:
```suggestion
```

**Review by klaasnicolaas — DISMISSED** [review]

**Review by frenck — APPROVED** [review]
Thanks, @dontinelli 👍 

../Frenck

**Pull request merged**